frontend/pages/news/article/create.tsx

```diff
@@ -77,10 +83,40 @@ export default function CreateArticlePage() {
   });
 
   const createArticle = async () => {
+    if (!header.sv) {
```

Comment:
These should be translated strings, especially since we're opening up news creation to all members.